app/src/main/res/values/options_values.xml

```diff
@@ -3,23 +3,26 @@
     <!-- Environments Options -->
     <string-array name="developer_options_environments" translatable="false">
         <item>@string/developer_options_env_void</item>
```

Comment:
this order look good to folks? I could do Void, Cave, Meadow, Offworld, Underwater, Winter instead. lmk.